org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/extensions/result/ContributionManager.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core.extensions.result;
+
+import java.util.List;
+
+/**
+ * Provides access to external plugin contributions.
+ *
+ * @author Donat Csikos
+ */
+public interface ContributionManager {
```

Comment:
We don't need a new interface here. We can reuse the existing InvocationCustomizer interface and have an implementation that collects all of them from the extension point.